src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -107,3 +134,39 @@ def init_sync_d_interface_tables():
         if_alias_map = dict(zip(if_name_map.keys(), if_name_map.keys()))
 
     return db_conn, if_name_map, if_alias_map, if_id_map, oid_sai_map, oid_name_map
+
+
+def init_sync_d_lag_tables(db_conn):
+    """
+    Helper method. Connects to and initializes LAG interface maps for SyncD-connected MIB(s).
+    :param db_conn: database connector
+    :return: tuple(lag_name_if_name_map, if_name_lag_name_map, oid_lag_name_map)
+    """
+    db_conn.connect(APPL_DB)
+
+    lag_entries = db_conn.keys(APPL_DB, b"LAG_TABLE:*")
+
+    # { lag_name (SONiC) -> [ lag_members (if_name) ] }
+    # ex: { "PortChannel0" : [ "Ethernet0", "Ethernet4" ] }
+    lag_name_if_name_map = {}
+
+    # { if_name (SONiC) -> lag_name }
+    # ex: { "Ethernet0" : "PortChannel0" }
+    if_name_lag_name_map = {}
+
+    for lag_entry in lag_entries:
+        lag_name = lag_entry[len(b"LAG_TABLE:"):]
+        lag_members = db_conn.keys(APPL_DB, b"LAG_MEMBER_TABLE:%s:*" % lag_name)
+
+        def member_name_str(val, lag_name):
+            return val[len(b"LAG_MEMBER_TABLE:%s:" % lag_name):]
+
+        lag_name_if_name_map[lag_name] = [member_name_str(m, lag_name) for m in lag_members]
```

Comment:
SNMP in SONiC works in read only mode. It takes information from DB converts it in special format and returns to user. Without working DB it is meaningless. If something bad happens with DB it will crash anyway because all code is implemented taking into consideration that DB can't fail. This is actual for any existing component.